### PR TITLE
[interp] remove last element from arrays and add const

### PR DESCRIPTION
**Pull request by jaykrell**
combine lines for searchability
add const

### CODE REVIEW

File: mono/mini/interp/mintops.c [hunk 1]
```diff
@@ -10,28 +10,22 @@
 #include <stdio.h>
 #include "mintops.h"
 
-#define OPDEF(a,b,c,d) \
-	b,
-const char *mono_interp_opname[] = {
+#define OPDEF(a,b,c,d) b,
+const char * const mono_interp_opname [] = {
```
Comment (jaykrell) [inline mono/mini/interp/mintops.c]: This will later change to array of short, i.e. half or quarter size.

File: mono/mini/interp/mintops.c [hunk 2]
```diff
@@ -10,28 +10,22 @@
...
-	0
 };
 #undef OPDEF
 
 
-#define OPDEF(a,b,c,d) \
-	d,
-MintOpArgType mono_interp_opargtype[] = {
+#define OPDEF(a,b,c,d) d,
+MintOpArgType const mono_interp_opargtype [] = {
```
Comment (jaykrell) [inline mono/mini/interp/mintops.c]: This shall later change to array of char, i.e. quarter size.

**Comment by jaykrell** [comment]
@monojenkins build failed

**Comment by vargaz** [comment]
Looks ok, not sure why does extra elements were there.

**Comment by jaykrell** [comment]
Maybe because of “trailing comma”?

**Comment by marek-safar** [comment]
@lewurm could you please review

**Pull request merged**